### DISCUSSION TITLE
Payment Declined. Updated Credit Card

### DISCUSSION THREAD

**Discussion by MetegrityIT** [community/community#118315] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello Community.

I encountered a problem with my credit card on file, which has since been resolved. However, despite updating the credit card details, GitHub's system will not process the payment. I attempted to rectify this by removing and re-adding the card, to no avail. Contacting phone support proved unhelpful, as they directed me to open a ticket, which I did two weeks ago. Unfortunately, I have yet to receive a response from support regarding the issue. As a result, 15 developers are unable to access Co-pilot. Is it normal for support to take this long to respond to a ticket? Two weeks appears excessively lengthy. Maybe this will get the attention of the support desk. Ticket #2698188.

**Comment by MetegrityIT** [comment]
How long does it take for support to respond to a ticket?

**Comment by balusena** [comment]
The delay in GitHub support responding to your ticket for two weeks is unusual. Continue following up on the ticket, ensuring all necessary information is provided. Consider reaching out through different support channels or escalating the issue to ensure prompt resolution for the impacted developers.

**Comment by MetegrityIT** [comment]
Back to top. Still no answer from support. Update ticket daily.

**Comment by asmawi117** [comment]
bro my ticket hasn't got any response after 2 months idk what is github support doink 😭

**Comment by ebndev** [comment]
Hi [MetegrityIT](https://github.com/MetegrityIT),

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
